src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonElement.cs

```diff
@@ -1476,7 +1476,10 @@ public bool ValueEquals(ReadOnlySpan<byte> utf8Text)
             if (TokenType == JsonTokenType.Null)
             {
                 // This is different than Length == 0, in that it tests true for null, but false for ""
-                return Unsafe.IsNullRef(ref MemoryMarshal.GetReference(utf8Text));
+                // TODO: Refactor as it is considered a bad practice to rely on null-ness of a Span._reference field.
```

Comment:
What issue is tracking this TODO? I'd prefer not to merge "TODO" comments that aren't tied to a particular issue.

And what would you expect the fix to be, given that it's already codified the difference into its behavior?

Reply:
I don't have a strong opinion nor ideas how to avoid it in STJ, I just wanted to make it clear that we don't promote this behaviour (for readers). Removed for now